k4MarlinWrapper/src/components/EDM4hep2Lcio.cpp

```diff
@@ -381,13 +381,13 @@ StatusCode EDM4hep2LcioTool::convertCollections(lcio::LCEventImpl* lcio_event) {
   std::vector<EDM4hep2LCIOConv::ParticleIDConvData> pidCollections{};
   std::vector<EDM4hep2LCIOConv::TrackDqdxConvData>  dQdxCollections{};
 
-  std::vector<std::tuple<std::string, const podio::CollectionBase*>> associations{};
+  std::vector<std::tuple<std::string, const podio::CollectionBase*>> linkCollections{};
 
   for (const auto& [edm4hepName, lcioName] : collsToConvert) {
     const auto coll = getEDM4hepCollection(edm4hepName);
-    if (coll->getTypeName().find("Association") != std::string_view::npos) {
```

Comment:
This is what #199 is referring to since now it's impossible to find "Association" in the type name.